pkgs/top-level/all-packages.nix

```diff
@@ -808,6 +810,12 @@ with pkgs;
 
   sea-orm-cli = callPackage ../development/tools/sea-orm-cli { };
 
+  vcpkg-tool = callPackage ../by-name/vc/vcpkg-tool/package.nix {
+    fmt = fmt_10;
+  };
+
+  vcpkg = callPackage ../by-name/vc/vcpkg/package.nix {  };
+
```

Comment:
```suggestion
```

Reply:
I thought the idea of the by-name namespace was that these packages will be included automatically in all-packages , why do we need to add them here when they don't add any attributes?